extensions/amp-analytics/0.1/vendors.js

```diff
@@ -198,6 +198,29 @@ export const ANALYTICS_CONFIG = {
     },
   },
 
+  'colanalytics': {
+    'vars': {
+        'id': '',
+        'colurl':'',
+    },
+    'requests': {
+        'host': 'https://${colurl}',
+        'base': '${host}/message',
+        'pageview': '${base}?cid=${id}&val_101=${canonicalPath}&val_101=${canonicalHost}&uuid=${uid}&val_101=${authors}&val_101=${zone}&val_101=${sponsorName}&val_101=${contentType}&val_101=${scrollTop}&val_101=${scrollHeight}&val_101=${decayTime}&val_101=${totalEngagedTime}&val_101=${documentReferrer}&val_101=${pageLoadTime}&val_108=${title}'
```

Comment:
Please note that using ${pageLoadTime} is going to delay the request till a value for that variable is available.